### PR TITLE
HRINT-356 Java Demos - Query Demos - Part 2

### PR DESCRIPTION
**Pull request by Danielle9897**

### CODE REVIEW

File: DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/queries/pagingQueryResults/metadata.json [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+                }
+            ]
+        },
+        {
+            "Title": "Skip results",
+            "Slug": "step-3",
+            "DescriptionLines": [
+                "* Use `skip()` to specify the number of results to skip from the total available result set.",
+                "",
+                "* The query above will _skip the first 'resultsToSkip' documents_.<br>",
```
Comment (ml054) [inline DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/queries/pagingQueryResults/metadata.json]: _skip - > _skip_ ?

File: DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/queries/projectingIndividualFields/metadata.json [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+                    "Type" : "link",
+                    "Url"  : "https://github.com/ravendb/ravendb-jvm-tutorials/blob/hospital/README.md#4-aggregating-and-projecting"
+                }
+            ]
+        },
+        {
+            "Title": "Define a projection",
+            "Slug": "step-2",
+            "DescriptionLines": [
+                "* Use `selectFields()` to project partial document data from the server.",
```
Comment (ml054) [inline DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/queries/projectingIndividualFields/metadata.json]: I wonder if we should mention about second overload - when we specify only target object class.

File: DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/queries/pagingQueryResults/metadata.json (hunk 1 above)
Comment (Danielle9897) [reply DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/queries/pagingQueryResults/metadata.json]: image

Comment (ml054) [reply DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/queries/pagingQueryResults/metadata.json]: ok

File: DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/queries/projectingIndividualFields/metadata.json (hunk 2 above)
Comment (Danielle9897) [reply DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/queries/projectingIndividualFields/metadata.json]: image

Comment (ml054) [reply DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/queries/projectingIndividualFields/metadata.json]: I wonder if sentense (When custom class fields are subset...) is necessary.

Like we talked it takes list and fills query data.

Comment (Danielle9897) [reply DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/queries/projectingIndividualFields/metadata.json]: removed that  line

**Pull request merged**